src/settings/ReadingRoomAccess/ReadingRoomAccess.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+
+import { TitleManager, useStripes } from '@folio/stripes/core';
+import {
+  Label,
+  Checkbox,
+} from '@folio/stripes/components';
+import { ControlledVocab } from '@folio/stripes/smart-components';
+
+const readingRoomsData = {
+  values: { records: [
+    {
+      'id': 1,
+      'name': 'RR1',
+      'public': true,
+      'servicePoint': [
+        {
+          name: 'Circ Desk 1',
+          id: '3a40852d-49fd-4df2-a1f9-6e2641a6e91f',
+        },
+        {
+          id: 'c4c90014-c8c9-4ade-8f24-b5e313319f4b',
+          name: 'Circ Desk 2'
+        },
+      ],
+      // metadata: {
+      //   'createdDate': '2024-03-21T10:59:25.085+00:00',
+      //   'createdByUserId': 'af5ad81e-6857-5b65-9c0c-60942e56f872',
+      //   'updatedDate': '2024-03-21T10:59:25.085+00:00',
+      //   'updatedByUserId': 'af5ad81e-6857-5b65-9c0c-60942e56f872'
+      // }
+    },
+    {
+      'id': 2,
+      'name': 'RR2',
+      'public': true,
+      'servicePoint': [{
+        name: 'Circ Desk 1',
+        id: '3a40852d-49fd-4df2-a1f9-6e2641a6e91f',
+      }],
+      // metadata: {
+      //   'createdDate': '2024-03-21T10:59:25.085+00:00',
+      //   'createdByUserId': 'af5ad81e-6857-5b65-9c0c-60942e56f872',
+      //   'updatedDate': '2024-03-21T10:59:25.085+00:00',
+      //   'updatedByUserId': 'af5ad81e-6857-5b65-9c0c-60942e56f872'
+      // }
+    },
+  ] },
```

Comment:
It is exceptionally exceptionally odd to me that this kind of data would be hard-coded and universal to every single tenant. The fact that `metadata` is included but commented out suggests this was copied from an API response; was this test data that should have been removed once the API was configured?